vmicore/src/lib/os/windows/KernelAccess.cpp

```diff
@@ -114,9 +114,15 @@ namespace VmiCore::Windows
         return currentListEntry - kernelOffsets.eprocess.ActiveProcessLinks;
     }
 
-    addr_t KernelAccess::extractDirectoryTableBase(addr_t eprocessBase) const
+    addr_t KernelAccess::extractKernelDirectoryTableBase(addr_t eprocessBase) const
     {
-        return vmiInterface->read64VA(eprocessBase + kernelOffsets.kprocess.DirectoryTableBase,
+        return vmiInterface->read64VA(eprocessBase + kernelOffsets.kprocess.kernelDirectoryTableBase,
```

Comment:
Can we always be sure that the lower bits are zero here or should we mask them as well? 🤔 My guess is that they should be zero, but I am not sure. Have you checked this?